Contacts/People/Council/council.md

```diff
@@ -27,19 +27,18 @@ with any of the Council members if you have information that we can use.
 
 In its role as editorial board, the GAP Council
 coordinates the process by which submitted GAP packages
-are refereed. In that sense, the Council chairman acts as editor in
+are refereed. In that sense, the Council chair acts as editor in
 chief, and the other members function as "associate editors," after the
 fashion of some journals. It is appropriate to obtain advice on submitting
-a package from any of the Council members. To submit a package, the Chair
-of the Council should be informed (by emailing
+a package from any of the Council members. To submit a package, the Council
+chair should be informed (by emailing
```

Comment:
I've changed this one spot from `Chair of the Council` to `Council chair`, but I wonder about the uppercase versus lower case "C".  This also appears twice on `Contacts/submit.html`, and I think it would be good to deal with this uniformly.

So, should it be `Council Chair` or `Council chair`? The former might make it even clearer that we are not talking about furniture... :-)